contracts/Staking.sol

```diff
@@ -392,100 +383,100 @@ contract Staking is Pauser, Whitelist {
     //     emit Slash(valAddr, request.nonce, slashAmt);
     // }
 
-    function collectForfeiture() external {
-        require(forfeiture > 0, "Nothing to collect");
-        CELER_TOKEN.safeTransfer(rewardContract, forfeiture);
-        forfeiture = 0;
-    }
+    // function collectForfeiture() external {
```

Comment:
Since we move the verification part of LightClient, should we undo the change on staking contract?

Reply:
Yes, we don't need to make any changes to the stake contract